docker/2diirunner-with-packages/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM maurolepore/2diirunner:1.3.0
+MAINTAINER "Mauro Lepore" maurolepore@gmail.com
+
+RUN apt-get update && apt-get install -y pandoc
+
+# This repository has binaries not just for Windows and Mac, but also for Linux
```

Comment:
actually, as far as I understand, this repo has binaries *only* for Linux... i.e. this will not benefit you if you try to use this on macOS or Windows

Reply:
I think the container runs on Linux, even if you run docker on a mac or windows.

Reply:
I was talking about the comment about RStudio's repository... their repository serves Linux binaries, but as far as I can tell it does not server Mac or Windows binaries.